inlong-sort/sort-connectors/elasticsearch-base/pom.xml

```diff
@@ -152,6 +152,11 @@
             <artifactId>log4j-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>audit-sdk</artifactId>
+            <version>1.3.0-SNAPSHOT</version>
```

Comment:
It is better to use ${xxx.version} to replace it.

Reply:
DONE